site/_docs/configuration.md

```diff
@@ -448,9 +448,8 @@ The `projects/foo_project.md` would have the `layout` set to `foobar` instead of
 
 ## Default Configuration
 
-Jekyll runs with the following configuration options by default. Unless
-alternative settings for these options are explicitly specified in the
-configuration file or on the command-line, Jekyll will run using these options.
+Jekyll runs with the following configuration options by default. Alternative settings for these options can be explicitly specified in the
```

Comment:
Oh, yeah, my bad. I have BBEdit set to wrap text at its "Page Guide" which is 72 characters.